### ISSUE TITLE
user limit reached 

### ISSUE THREAD

**Issue by dseixas** [LibrePlan/libreplan#1925]
Hi, 

I've finally managed to get libreplan up and running. But when I try to create a new database user, the button for adding is disabled and showing the "User limit reached" message.

Why is this happening? I've checked the database and the limit is way beyond the 8 default users.

Thanks

**Comment by kwoot** [comment]
You could try a value of zero, or deleting the user limit record. This feature as developed for cloud users of LibrePlan. It is not meant to be a nuisance.

**Comment by dseixas** [comment]
I had it previously configured at 0. But it had the same limit.

Also I have the same problem with 10 resources limit, no matter what I
write down on the database.

Is there a "cloud" installation identifier?

**Comment by kwoot** [comment]
No, there isn't . There are only a few optional records to facilitate possible cloud installations to limit users, resources (workers and/or machines). I need to document this, I know.

**Comment by dseixas** [comment]
Well, just a quick note. 

I haven't been able to remove the restriction, but FWIW I can arrange a skype or similar meeting so you can have a look at my libreplan.
